deploy/kubernetes/gcloud-dev-settings.yaml

```diff
@@ -32,6 +32,10 @@ ES_DATA_DISK_TYPE: pd-standard
 ES_DATA_DISK_SIZE: 10
 ES_POD_VOLUME_SIZE: '1Gi'
 
+ES_CONFIGURE_SNAPSHOTS: true
```

Comment:
I think generally we want this turned off for dev, so I would say we want to switch this before merging